### ISSUE TITLE
[Upload save] COD WaW

### ISSUE THREAD

**Issue by Lore11-93** [bucanero/apollo-saves#403]
### Game name

Call of Duty: World at War

### Title ID

BLES00354

### Save description

SAVESET (NO DLC) Part 4
[CALL OF DUTY® WORLD AT WAR (COMPLETED).zip](https://github.com/bucanero/apollo-saves/files/14323046/CALL.OF.DUTY.WORLD.AT.WAR.COMPLETED.zip)

### FW Version

None

**Comment by bucanero** [comment]
hello,
could you explain better these ps3 saves?

it seems is a completed save for Call of duty world at war, but I'm not sure why there are 4 different parts / saves.

providing additional details can help so they can be added correctly

**Comment by Lore11-93** [comment]
Yes, of course. The files contain several saves for different CoD WaW trophies, there were too many to load them all in one file.
